packages/store/tests/select.spec.ts

```diff
@@ -231,7 +234,7 @@ describe('Select', () => {
   }));
 
   it('should throw an exception if reserved key used in class', () => {
```

Comment:
This description of the expectation does not match the assertion at the end anymore.